bootstrap.py

```diff
@@ -688,7 +708,10 @@ class MEOptions:
 
 
 if __name__ == '__main__':
-
+    
+    # Check if Satellite or Capsule are installed or not
+    check_prerequisite()
```

Comment:
Might want to put this after options parsing. (basically after [here](https://github.com/Katello/katello-client-bootstrap/blob/master/bootstrap.py#L866). No need in checking until we have a sane set of options.  Also, you get an error when running with no options. 

~~~
./bootstrap.py 
Foreman Bootstrap Script
This script is designed to register new systems or to migrate an existing system to a Foreman server with Katello
[ERROR], [2017-03-15 10:52:59], EXITING: [RPM Satellite/capsule installed] failed to execute properly.
Must specify server, login, organization, hostgroup and activation key.  See usage:
Usage: bootstrap.py [options]


~~~